spec/models/article_spec.rb

```diff
@@ -1091,6 +1091,16 @@ def foo():
     end
   end
 
+  describe "#followers" do
+    it "returns an array of users who follow the article's author" do
+      following_user = create(:user)
+      following_user.follow(user)
+
+      expect(article.followers.length).to eq(1)
+      expect(article.followers.first.username).to eq(following_user.username)
```

Comment:
Another benefit of extracting this onto the `Article` model: added a quick spec! ✅